scripts/eval/README.md

```
@@ -0,0 +1,306 @@
+# Verbose output
+node scripts/eval/eval.ts -p mealdrop --prompt pattern-copy-play -v
+
+# List available projects, models, or prompts
+node scripts/eval/eval.ts --list-projects
+node scripts/eval/eval.ts --list-models
+node scripts/eval/eval.ts --list-prompts
+```
+
+When a trial completes, it prints a summary:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)